account-decoder/src/parse_config.rs

```diff
@@ -66,6 +68,10 @@ pub struct UiConfigKey {
     pub signer: bool,
 }
 
+#[deprecated(
+    since = "1.14.21",
```

Comment:
Let's not backport this to 1.14
```suggestion
    since = "1.16.7",
```